docs/ENTITY_RELATIONSHIP_DIAGRAM.md

```diff
@@ -11,15 +11,16 @@ erDiagram
   }
   Applicant {
     int id
-    string externalId 
+    string externalId
     bool hasGivenPermission
   }
   ExternalSource {
     int id
     string name
   }
-  Application {
+  Proposal {
     int id
+    string externalId
```

Comment:
What is the reasoning behind the redundant externalId? One externalId is already present via `int applicationId` below.

Reply:
Ah, I see, it is not the same as the other externalId at all, it is a proposal external ID, not an applicant external ID.

Reply:
This is the external ID for the proposal (e.g. not `applicantExternalId`, not the applicant (unless I'm misunderstanding the question).  I didn't add `proposal` to the field name since it's already in the table.  Does that make sense?

Reply:
Yes it makes sense. In the same way that we do not have `applicants.applicantId` or `proposals.proposalId` but rather `applicants.id` and `proposals.id` we have `applicants.externalId` and `proposals.externalId`.